stripe-ui-core/src/main/java/com/stripe/android/uicore/strings/ResolvableStringComposeUtils.kt

```diff
@@ -0,0 +1,14 @@
+package com.stripe.android.uicore.strings
+
+import androidx.annotation.RestrictTo
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.ReadOnlyComposable
+import androidx.compose.ui.platform.LocalContext
+import com.stripe.android.core.strings.ResolvableString
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+@Composable
+@ReadOnlyComposable
+fun resolvableStringResource(resolvableString: ResolvableString): String {
```

Comment:
Nice! I was about to suggest this in another comment until I saw this 🥳

Nit: We _could_ change this to `fun ResolvableString.resolve(): String` to make the call site a bit more succinct.